tests/catch/unit/stream/hipStreamDestroy.cc

```diff
@@ -27,12 +27,14 @@ TEST_CASE("Unit_hipStreamDestroy_Default") {
   HIP_CHECK(hipStreamDestroy(stream));
 }
 
+#if !HT_NVIDIA
```

Comment:
Please put a comment why this testcase is disabled for AMD platform. If there is JIRA defect raised for this, then please mention the JIRA defect id in the comment.

Reply:
@rorake  When retested, testcase does not provoke an error on NVIDIA platform. Addressed issues are fixed.